cmd/osbuild-image-tests/constants/constants.go

```diff
@@ -34,7 +34,7 @@ var TestPaths = struct {
 	MetaData                string
 	AzureDeploymentTemplate string
 }{
-	ImageInfo:               "/usr/libexec/osbuild-composer-test/image-info",
+	ImageInfo:               "/usr/sbin/osbuild-image-info",
```

Comment:
Should you remove the prefix here as well? If not, I'm sorry, but I dropped the PR to move the script to /sbin, since Fedora is going to merge /bin and /sbin anyway. :innocent:

Reply:
Changed to plain `osbuild-image-info`.